titapapunne/MoreThanWords
# #20: Real User Data and Total Outcomes in Trainee Pod Screen

- Added to the outcomes endpoint in app.py to calculate the total number of outcomes and then used that info in the frontend of TraineePod.js for the progress bars
- Change to using real user data instead of fake user data by passing in the user parameter in app.py and by using token in TraineePod.js (also changed from axios to fetch).

**Steps to test PR:** 
- Login in on app using Yichen's account to test having real user data: 
Username: my.waymao@gmail.com
password: ZijReFQ@AJYz3jfrA3iz

- Then after clicking on Trainee Pod from Home Screen, should be able to load all the progress bars with the correct number of total outcomes and completed outcomes.

Can do "print(total_and_completed_outcomes)" in app.py to see all the dictionary with total outcomes info

Standard Checklist:
- [ x ] Does the code work? Does it perform its intended function, the logic is correct etc.
- [ x ] Have you merged the most recent version of staging into your branch?
- [ x ] Are all merge conflicts resolved? 
- [ x ] Can any logging or debugging code be removed? (Any debugging statements should be removed)
- [ x ] Is the `diff` the **minimum amount** needed to achieve this ticket's goal?
